spec/javascript/components/applications/__snapshots__/ListingApplicationsPage.test.js.snap

```diff
@@ -201,7 +201,7 @@ Array [
                   <option
                     value=""
                   >
-                    Select One...
```

Comment:
The type field used to default to "Select One..." with "Any Type" at the bottom of the list. I considered this a bug since select one and any type have the same functionality, so now we just default to "Any Type"

## Screenshot (before)
<img width="223" alt="- 2020-05-12 at 2 24 29 PM" src="https://user-images.githubusercontent.com/64036574/81747174-78a1e780-945c-11ea-8c39-074eec665bc5.png">

## Screenshot (after)
<img width="224" alt="- 2020-05-12 at 2 24 10 PM" src="https://user-images.githubusercontent.com/64036574/81747217-8ce5e480-945c-11ea-9465-6658ea3b8cfe.png">